### ISSUE TITLE
Attach build to GitHub releases

### ISSUE THREAD

**Issue by kumy** [olvid-io/olvid-android#3]
As said in https://github.com/olvid-io/olvid-android/issues/2#issuecomment-1004001931, it would be awesome to have already built packages signed by your own key attached to the GitHub releases. (it would also be nice to have the download apk link on your website)

This would help:
- from having to build it manually
- for having it included in alternatives repo such as [F-droid IzzySoft](https://gitlab.com/IzzyOnDroid/repo/-/issues/221)
- would permit to enable [push notification](https://github.com/olvid-io/olvid-android#getting-google-push-notifications)(?)

Thanks a lot!

**Comment by kumy** [comment]
New builds `v0.13.3_195`
https://github.com/kumy/olvid-android/releases/tag/v0.13.3_195-kumy1

**Comment by finiasz** [comment]
Hello @Morse6690, we know several users would be interested in an official "non store" APK but we do not want to incentivize them to install APKs signed by a random key. Also, it would be easy for anyone impersonating the github website to distribute a malicious APK.

At least, F-Droid and Google Play can check the APK is properly signed.

**Comment by Morse6690** [comment]
I can understand your concerns with this. Having said that there is a use case for this, hence apps like signal make sure they provide an apk that can be downloaded from outside the play store.
People who can't access the play store can either use a dodgy third party website like apkpure or similar or they just don't install it. I would think these third party app sites have far greater risk to an end user than an app placed by the developer on github, or hosted  on your website in a non obvious location, similar to what signal has done.
Of course these users cannot use the payment features of the play store to pay for the extras features.
However I believe that the users who may be interested enough to download via this method are savvy enough understand the dangers. The average person will only ever use the play store.

**Comment by Morse6690** [comment]
Is it possible to do something similar to signal? 
They provide an apk that can be downloaded directly from their own website. Users are prompted when a new update is available which can be installed by hitting "ok" on the "new update available" notification.

**Comment by finiasz** [comment]
Just a quick note here: we are now providing an APK for download directly from our website. This APK is a universal APK signed with the same keys as the version available on Google Play. This means push notifications can work for users with Google Services installed. We do not provide an APK for the "no google" flavor of the app.

https://olvid.io/download/

**Comment by kumy** [comment]
That's a great step forward, thanks. What are the reason behind the "We do not provide an APK for the "no google" flavor of the app."?

**Comment by Morse6690** [comment]
Making sure the "use foreground service" option is turned on should mean that notifications will be received whether the device has Google play services installed or not?
Is that right?

**Comment by Morse6690** [comment]
That's right... Forgot that other setting. I have used those two settings on my Graphenos device for 18 months. The battery drain was pretty minimal, and never had a problem receiving messages.

**Comment by kumy** [comment]
For those like me using the "no google" version, I'll continue to build the releases on my side.
Here is [v1.3_220-kumy1](https://github.com/kumy/olvid-android/releases/tag/v1.3_220-kumy1)
